driver/sanitizer_hooks_with_pc.cpp

```diff
@@ -36,6 +36,20 @@
 
 #define REPEAT_4096(a) REPEAT_16(REPEAT_16(REPEAT_16(a)))
 
+// The first four registers to pass arguments in according to the
```

Comment:
Should we mention in the docs that we only support X64 platforms?

Reply:
We currently say `Jazzer supports x64 Linux and (experimentally) macOS 10.15.`. But good point, we should update this for Windows.